lib/class-wp-widget-block.php

```diff
@@ -197,6 +197,7 @@ public function form( $instance ) {
 		$instance = wp_parse_args( (array) $instance, $this->default_instance );
 		?>
 		<p>
+		/* translators: HTML code of the block, not an option that blocks HTML */
```

Comment:
Thanks for the PR!

I don't think this exact code would work, as we're inserting a PHP comment inside an HTML `<p>` tag.

We can also replace `echo __()` with `_e()` while we're at it.

I would suggest something like this to match a similar change for WordPress core in [r51814](https://core.trac.wordpress.org/changeset/51814):
```
<label for="<?php echo $this->get_field_id( 'content' ); ?>">
	<?php
	/* translators: HTML code of the block, not an option that blocks HTML. */
	_e( 'Block HTML:' );
	?>
</label>
```